rudder-core/src/main/scala/com/normation/rudder/services/queries/DynGroupUpdaterService.scala

```diff
@@ -104,11 +104,7 @@ class DynGroupUpdaterServiceImpl(
       newMemberIdsSet = newMembers.map( _.id).toSet
       savedGroup <- {
                       val newGroup = group.copy(serverList = newMemberIdsSet)
-                      if(group.isSystem) {
-                        woNodeGroupRepository.updateSystemGroup(newGroup, modId, actor, reason)
-                      } else {
-                        woNodeGroupRepository.update(newGroup, modId, actor, reason)
-                      }
+                      woNodeGroupRepository.updateDynGroupNodes(newGroup, modId, actor, reason)
```

Comment:
Only the udpate for the faulty group. I think it is ok in that case to wait for the next update (we don't even know if it's still a dyngroup, and it was updated just right now, so the period for update is consistant if we wait)